tests/harness.go

```diff
@@ -24,16 +25,27 @@ func main() {
 	log.Println(len(testNames), " tests to run")
 	buildFakeTerraformRegistry()
 	destroyMode := os.Getenv("DESTROY_MODE")
+
+	var wg sync.WaitGroup
```

Comment:
Nothing fancy. Running each test inside a go-routine.
Waits for all tests to finish.
If a test fails halts and exits (all tests stop).